extensions/notebook/src/jupyter/jupyterController.ts

```diff
@@ -175,13 +175,13 @@ export class JupyterController implements vscode.Disposable {
 	}
 
 	private async handleDependenciesReinstallation(): Promise<void> {
-		if (await this.confirmReinstall()) {
-			this._jupyterInstallation = await JupyterServerInstallation.getInstallation(
-				this.extensionContext.extensionPath,
-				this.outputChannel,
-				this.apiWrapper,
-				undefined,
-				true);
+		try {
+			let doReinstall = await this.confirmReinstall();
+			if (doReinstall) {
+				await this._jupyterInstallation.startInstallProcess(true);
+			}
+		} catch (err) {
+			this.apiWrapper.showErrorMessage(utils.getErrorMessage(err));
```

Comment:
Given we no longer reject the promise, what's the impact? This fixes something? Maybe add a comment explaining what'll happen thanks to this change.

Reply:
I started seeing errors that only said the Reinstall Dependencies command had failed, not the underlying rejection error message. So I changed it to print the error directly.

Reply:
Sounds good, thanks for clarifying/